modules/mirror-registry-localhost.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-installation-images.adoc
+
+[id="mirror-registry-localhost{context}"]
+= Mirroring on a local host with mirror registry for {product-title}
+
+This procedure introduces how to install the mirror registry on a local host using the `mirror-registry` installer tool. By doing so, users can create a local host registry running on port 443 for the purpose of storing a mirror of {product-title} images.
+
+[NOTE]
+====
+Installing Red Hat Quay using the `mirror-registry` CLI tool makes several changes to your machine. After installation, a `/etc/quay-install` directory is created, which has installation files, local storage, and the configuration bundle. Trusted SSH keys are generated in case the deployment target is the local host, and systemd files on the host machine are set up to ensure that container runtimes are persistent. Additionally, an initial user named `init` is created with an auto-generated password. All access credentials are printed at the end of the install routine.
+====
+
+.Procedure
+
+. Download the `mirror-registry.tar.gz` package for the latest version of mirror registry found on the link:https://console.redhat.com/openshift/downloads[*Downloads*] page of {product-title}.
+
+. Install the mirror registry on your local host with your current user account by using the `mirror-registry` tool:
++
+[source,terminal]
+----
+$ sudo ./mirror-registry install \
+  --quayHostname <host_example_com> \
+  --quayRoot <example_directory_name>
+----
```

Comment:
Lets add notes below this that point back to https://github.com/openshift/openshift-docs/pull/39593/files#diff-69cfe3d2263e019a7a1c1e939ecd40e84d804eff718d709a4da103d244f1792f and show users what options they can set; to over additional configuration during install.